emailer/src/emailer.js

```diff
@@ -27,7 +32,54 @@ const getInactiveApplications = async () => {
 
   await client.end()
 
-  return result
+  return result.rows
+}
+
+try {
+  let root, mailConfig
+  if (productionEnv) {
+    root = process.env.SITE_URL
+    mailConfig = {
+      host: 'smtp.sendgrid.net',
+      port: 587,
+      pool: true,
+      secure: false,
+      auth: {
+        user: 'apikey',
+        pass: process.env.SENDGRID_APIKEY
+      }
+    }
+  } else {
+    root = 'http://localhost:3000'
+    mailConfig = {
+      //logger: true,
```

Comment:
Is this needed? If not, it should be removed or commented somehow, e.g. 'When debugging and need more logging, uncomment this'